### ISSUE TITLE
[Bug]: collapsible_controls doesn't work when state is "standby"

### ISSUE THREAD

**Issue by CarterBrehm** [piitaya/lovelace-mushroom#535]
### Current Behavior

Currently, setting `collapsible_controls: true` on a mushroom media player card doesn't collapse the controls when the state of the media_player is `standby`.

### Expected Behavior

Since `standby` is equivalent to off, the controls do not work and should instead be hidden, as if the media_player was `off`.

### Steps To Reproduce

My environment is a Insignia Roku TV, connected to home assistant using the built-in [Roku integration](https://www.home-assistant.io/integrations/roku/). It never reports the TV as `off`, so the collapsible controls never function.

### Environment

```markdown
- OS: Ubuntu Server 16.04
- Browser: Safari/Firefox (tested on both, no change)
- Frontend Version: 20220531.0 - latest
- Core Version: 2022.6.0
```

**Comment by KTibow** [comment]
Is it still standby once the Roku is turned on?

**Comment by CarterBrehm** [comment]
No, once the Roku TV is on, it behaves as normal:

<img width="396" alt="CleanShot 2022-07-30 at 14 04 19@2x" src="https://user-images.githubusercontent.com/26262673/181941076-5b8876b3-1caf-4bd6-b4d7-e21ee9b9599a.png">

<img width="395" alt="CleanShot 2022-07-30 at 14 04 36@2x" src="https://user-images.githubusercontent.com/26262673/181941192-414164b8-c1f5-4197-aa57-2ef73bdb9e32.png">

**Comment by CarterBrehm** [comment]
I think all that needs to be changed is the hardcoded values that dictate when the collapsible controls show. For example, on [line 109 in utils.ts](https://github.com/piitaya/lovelace-mushroom/blob/0311fcdc7e601478e2333cee069fd0e2f2f4610f/src/cards/media-player-card/utils.ts), there's a line that checks if the control should show by seeing if `state === off`, I think this is solved by changing that to a simple `state === off || state === standby`.

**Comment by piitaya** [comment]
It will be fixed with the 2.3.1 version available tomorrow with HA 2022.11 release
